src/string.cr

```diff
@@ -2402,6 +2402,25 @@ class String
     true
   end
 
+  # Returns `self` unless `#blank?` is `true` in which case it returns `nil`.
+  #
+  # ```
+  # "a".presence || "default" # => "a"
```

Comment:
Please remove `|| "default"` here. The first examples should be as minimal as possible. Having it in the hash example is enough.

Reply:
Done.